sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session_receiver.py

```diff
@@ -45,9 +48,9 @@ class ServiceBusSessionReceiver(ServiceBusReceiver, SessionReceiverMixin):
      automatically shutdown. The default value is 0, meaning no timeout.
     :keyword mode: The mode with which messages will be retrieved from the entity. The two options
      are PeekLock and ReceiveAndDelete. Messages received with PeekLock must be settled within a given
-     lock period before they will be removed from the queue. Messages received with ReceiveAndDelete
-     will be immediately removed from the queue, and cannot be subsequently rejected or re-received if
-     the client fails to process the message. The default mode is PeekLock.
+     lock period (or have their lock renewed) before they will be removed from the queue. Messages received 
```

Comment:
```suggestion
     lock period before they will be removed from the queue. Messages received 
```

Might be just me but the additional paren-text made it seem like lock _renewal_ was somehow related to the message being removed. It read clearer without it (and the users have plenty of other places to understand lock renewal).